frontend-react/src/config/endpoints/messageTracker.ts

```diff
@@ -0,0 +1,22 @@
+export interface QualityFilter {
```

Comment:
we literally had the same idea as i had to add the endpoints for the message search ui! :laughing:
we're going to have merge conflicts but it'll be easy to resolve :smile: